docs/spec/Error.schema.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "Error",
     "properties": {
-        "error": { "$ref": "ErrorCode.schema.json" },
+        "error": { "type": "string" },
```

Comment:
porcelain doesn't restrict this

Reply:
Does that mean we can lose `ErrorCode.schema.json`?